qa/common/gen_qa_models.py

```diff
@@ -728,8 +728,15 @@ def create_plan_dynamic_modelfile(models_dir, max_batch, model_version,
             profile[i].set_shape("INPUT1", min_shape, opt_shape, max_shape)
         else:
             bs = [ max_batch - i if max_batch > i else 1]
-            profile[i].set_shape("INPUT0", [1] + min_shape, bs + opt_shape, bs + max_shape)
-            profile[i].set_shape("INPUT1", [1] + min_shape, bs + opt_shape, bs + max_shape)
```

Comment:
@tanmayv25 I changed the max dims here to be `[max_batch, max_shape]` so that all profiles are aligned with the model config, is that okay? I think `bs + max_shape` is typo.

Reply:
It was done purposely to catch our shape validation logic.

Reply:
Yeah, noticed and reverted the change on max_dims